Susi/Controllers/SignUpController/SignUpVCMethods.swift

```diff
@@ -129,7 +129,7 @@ extension SignUpViewController {
                 emailTextField.dividerActiveColor = .green
             }
         } else if textField == passwordTextField, let password = passwordTextField.text {
-            if password.isEmpty || password.characters.count < 5 {
+            if password.isEmpty && password.characters.count < 5 {
```

Comment:
Hi,
Why replace `||` with `&&`?

Reply:
Using && if both our condition gets true then we can proceed further
While using or || if one of the condition gets true we can proceed but it's not much secure and password strength is also not good
We can also strength the password by applying condition that the first letter of our password should be uppercase and its contain one special character also

Reply:
Here, the statement can be reduced to just checking the character length.
`if password.characters.count < 5` right?

Reply:
Yes right.

Reply:
Please make changes and commit.